serializer-deserializer/pom.xml

```diff
@@ -101,11 +101,44 @@
             <artifactId>wire-schema</artifactId>
             <version>${square-wireschema.version}</version>
         </dependency>
+
+<!--        TODO: clean up the dependency once Apicurio releases the latest version with the json_name fix -->
```

Comment:
Minor: Please remove it completely. We can always go back in git history to get this.